docs/server/source/drivers-clients/websocket-event-stream-api.rst

```diff
@@ -0,0 +1,105 @@
+The WebSockets Events Stream API
+================================
+
+BigchainDB provides real-time event streams over the WebSockets protocol with
+the Events Stream API.
+
+Connecting to a Events Stream from your application enables a BigchainDB node
+to notify you as events are processed, such as new `validated transactions <#valid-transactions>`_.
+
+
+Determining Support for the Events Stream API
+---------------------------------------------
+
+In practice, it's a good idea to make sure that the node you're connecting with
+has support for the Events Stream API. To do so, send a HTTP GET request to the
+node's :ref:`Root URL <bigchaindb-root-url>` and check that the response
+contains a ``streams_<version>`` property in ``_links``::
+
+    {
+      "_links": {
+         "streams_v1": "ws://example.com:9984/api/v1/streams/"
+      }
+    }
+
+
+Connection Keep Alive
```

Comment:
> I wanted to eagerly close connections (if the client doesn't ping us back) to avoid leaving dead connections open.

:+1: 

> Not sure how easy this is to actually implement, and is something we can descope during the implementation (or remove if it's not a concern).


I think it's good to outline this behavior specifically in the spec here as it outlines a useful requirement for the implementation phase later on. How this will then be implemented is TBD (when we implement). It could turn out that some Python WebSockets implementations in fact implement this behavior already (they probably should as the WebSockets ietf rfc also talks about this behavior: https://tools.ietf.org/html/rfc6455#section-5.5.2).

For the implementation, I'd say we should push this as close to the websocket protocol as possible.
For this spec, this note is appropriate.